DEFAULT_CONFIG.json5

```diff
@@ -184,6 +184,22 @@
     },
   },
 
+  //  /// Overwrite default builders for specific key expressions
```

Comment:
I'd rather have :
```
publishers: {
    default_builders: [
        {
             key_exprs: ["demo/**", "example/key"],
             config: {
                    congestion_control: "Block",
                    encoding: "zenoh/example",
                    priority: "data_high",
                    express: true,
                    reliability: "best_effort",
                    allowed_destination: "remote",
             },
         },
     ],
}
```

Reply:
Agreed, it seperates the keyexprs from the config, which is a lot more readable.